analyzer/src/funTest/assets/projects/synthetic/pip-python3/.gitignore

```diff
@@ -26,3 +26,6 @@ pip-log.txt
 #Mr Developer
 venv
 *.pyc
+
+# python-inspector cache directory
+.cache/
```

Comment:
Uh, `.cache` is a rather generic name. If this is a directory name that the creators of *python-inspector* came up with, is there a chance to change it, like to `.python-inspector`? Or does this name come from some third-party Python tooling that *python-inspector* makes use of?

Reply:
I have created an issue for this: https://github.com/nexB/python-inspector/issues/63

Reply:
Also, instead of adding the directory to `.gitignore` I am deleting it now for reasons explained in the first commit of this PR.